lib/modules/showImages.js

```diff
@@ -56,8 +57,8 @@ module.options = {
 	},
 	preloadImages: {
 		type: 'boolean',
-		value: false,
-		description: 'Preload gallery images for faster browsing. Beware: this is at the expense of lots of bandwidth usage.',
+		value: true,
+		description: 'Preload 2 gallery pieces for faster browsing.',
```

Comment:
Modified default here since new gallery behavior limits number of preloaded pieces.